dmroles.tex

```diff
@@ -90,6 +90,12 @@ \subsection{Pipelines Scientist \label{role:pipe}}
 
 \end{itemize}
 
+\subsection{Science Platform Scientist \label{role:scip}}
+The science platform is composed of three aspects. Each aspect is produced in a different institution.
```

Comment:
Capitalize as "Science Platform".
Place quotation marks around first use of "aspects" to indicate that it is a quasi-definition of the term in this context.